src/Symfony/Component/Notifier/Bridge/Telegram/TelegramOptions.php

```diff
@@ -156,4 +166,164 @@ public function answerCallbackQuery(string $callbackQueryId, bool $showAlert = f
 
         return $this;
     }
+
+    /**
+     * @return $this
+     */
+    public function location(float $latitude, float $longitude): static
+    {
+        $this->options['location'] = ['latitude' => $latitude, 'longitude' => $longitude];
+
+        return $this;
+    }
+
+    /**
+     * @return $this
+     */
+    public function venue(float $latitude, float $longitude, string $title, string $address): static
+    {
+        $this->options['venue'] = [
+            'latitude' => $latitude,
+            'longitude' => $longitude,
+            'title' => $title,
+            'address' => $address,
+        ];
+
+        return $this;
+    }
+
+    /**
+     * @return $this
+     */
+    public function document(string $url): static
+    {
+        $this->options['document'] = $url;
+
+        return $this;
+    }
+
+    /**
+     * @return $this
+     */
+    public function uploadDocument(string $path): static
+    {
+        $this->options['upload']['document'] = $path;
+
+        return $this;
+    }
+
+    /**
+     * @return $this
+     */
+    public function video(string $url): static
+    {
+        $this->options['video'] = $url;
+
+        return $this;
+    }
+
+    /**
+     * @return $this
+     */
+    public function uploadVideo(string $path): static
+    {
+        $this->options['upload']['video'] = $path;
+
+        return $this;
+    }
+
+    /**
+     * @return $this
+     */
+    public function audio(string $url): static
+    {
+        $this->options['audio'] = $url;
+
+        return $this;
+    }
+
+    /**
+     * @return $this
+     */
+    public function uploadAudio(string $path): static
+    {
+        $this->options['upload']['audio'] = $path;
+
+        return $this;
+    }
+
+    /**
+     * @return $this
+     */
+    public function animation(string $url): static
+    {
+        $this->options['animation'] = $url;
+
+        return $this;
+    }
+
+    /**
+     * @return $this
+     */
+    public function uploadAnimation(string $path): static
+    {
+        $this->options['upload']['animation'] = $path;
+
+        return $this;
+    }
+
+    /**
+     * @return $this
+     */
+    public function sticker(string $url, string $emoji = null): static
+    {
+        $this->options['sticker'] = $url;
+        if (null !== $emoji) {
+            $this->options['emoji'] = $emoji;
+        } else {
+            if (isset($this->options['emoji'])) {
+                unset($this->options['emoji']);
+            }
+        }
```

Comment:
what about removing all those if/else from the class (aka just do `$this->options['emoji'] = $emoji;) and add an array_filter in the class that turns the options into a URL?